src/main/java/tqs/example/impostor/repository/Admin.java

```diff
@@ -0,0 +1,65 @@
+package tqs.example.impostor.repository;
+
+import jakarta.persistence.*;
+
+import java.util.Objects;
+
+@Entity
+@Table(name = "ADMIN")
+public class Admin {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "userName")
+    private String userName;
+
+    @Column(name = "password")
```

Comment:
Those columns could be set as nullable = false, to make sure that every admin has a username and password.